modules/build/src/main/scala/scala/build/Inputs.scala

```diff
@@ -262,7 +264,7 @@ object Inputs {
   }
 
   private val githubGistsArchiveRegex: Regex =
-    s""":\\/\\/gist\\.github\\.com\\/[^\\/]*?\\/[^\\/]*$$""".r
+    s"""gist\\.github\\.com\\/[^\\/]*?\\/[^\\/]*$$""".r
```

Comment:
That means, it also matches local directories with the same pattern, right? Maybe with proper gist-like zip support as input, we could get rid of that (and only match actual gist URLs here). But I guess this can be fixed in another PR.

Reply:
Right, it matches also local directories. I also think it can be done in a separate PR.